packages/front-end/components/Settings/MemberList.tsx

```diff
@@ -72,28 +82,42 @@ const MemberList: FC<{
               <td>
                 {member.id !== userId && (
                   <>
-                    <a
-                      href="#"
-                      className="tr-hover mr-3"
-                      onClick={(e) => {
-                        e.preventDefault();
-                        setRoleModal(member);
-                        setRole(member.role);
-                      }}
-                    >
-                      <FaPencilAlt />
-                    </a>
-                    <DeleteButton
-                      link={true}
-                      className="tr-hover"
-                      displayName={member.email}
-                      onClick={async () => {
-                        await apiCall(`/member/${member.id}`, {
-                          method: "DELETE",
-                        });
-                        mutate();
-                      }}
-                    />
+                    <MoreMenu id="test">
+                      <button
+                        className="dropdown-item pe-auto"
```

Comment:
We're using Bootstrap 4.6 which doesn't have classes for pointer events, so `pe-auto` doesn't do anything.